README.md

```diff
@@ -16,9 +16,8 @@ It also supports the diagnostic metrics:
 * [Time to First Byte](https://web.dev/articles/ttfb)
 * [First Contentful Paint](https://web.dev/articles/fcp)
 
-And finally, it temporarily supports the following deprecated metrics:
-
-* [First Input Delay](https://web.dev/articles/fid)
+> [!CAUTION]
+> [First Input Delay](https://web.dev/articles/fid) is deprecated and no longer supported by Chrome.
```

Comment:
Do we really need this? If anyone asks we can point to the post but having this hear feels like a bad smell of dead things.